### DISCUSSION TITLE
Campaign order in fheroes2

### DISCUSSION THREAD

**Discussion by TLCD17** [ihhub/fheroes2#9059] [Q&A] [answered]
### Your question

What is the order of campaigns in fheroes2? Which is first, which is second, which is third, and which is last?
And in the main (original) campaigns of fheroes2, is there a campaign one and two, or one and no more?

**Comment by zenseii** [comment]
I'm changing this to discussion because it regards info about the original game.

**Comment by LeHerosInconnu** [comment, marked as answer]
Hello @TLCD17,

There is no real order to play the expansion campaigns (The Price of Loyalty) or the original game campaigns (The Succession Wars).
There are two independent campaigns in the original game.
Players can play them in any order they wish. :)

**Reply by zenseii** [reply]
Yep, the first campaign in order is either Archibald's or Roland's campaign. Some don't know that these are two different campaigns with mostly different scenarios and plot.

As for difficulty curve, you can pick either and play on Easy difficulty if you're new to the game. Just note that you can't turn up the difficulty after you've started unless you restart.

Meanwhile, for the difficulty curve for the Price of Loyalty expansion campaigns, I suppose going from easiest to hardest I would rank them like this: Voyage Home -> Descendants -> Wizard's Isle -> The Price of Loyalty, though this is down to personal opinion.
